android/src/main/java/com/reactlibrary/Helpers.kt

```diff
@@ -16,6 +16,21 @@ import com.judopay.judokit.android.model.googlepay.GooglePayBillingAddressParame
 import com.judopay.judokit.android.model.googlepay.GooglePayEnvironment
 import com.judopay.judokit.android.model.googlepay.GooglePayShippingAddressParameters
 
+// For consistency with:
+// https://github.com/Judopay/JudoKit-iOS/blob/abd34bbfe4784fb5f074ed30f93d6743ba295622/Source/Models/Response/JPResponse.m#L32
+private const val STATUS_DECLINED = "declined"
+private const val STATUS_SUCCESS = "success"
+private const val STATUS_ERROR = "error"
+
+// For consistency with:
+// https://github.com/Judopay/JudoKit-iOS/blob/abd34bbfe4784fb5f074ed30f93d6743ba295622/Source/Models/Transaction/JPTransactionResult.h#L27
```

Comment:
Might worth updating the url to 'master` instead of the commit sha - eg:
https://github.com/Judopay/JudoKit-iOS/blob/master/Source/Models/Transaction/JPTransactionResult.h#L27
same on lines 20 and 73